ciao-controller/main.go

```diff
@@ -136,6 +136,8 @@ func main() {
 	identityURL = clusterConfig.Configure.IdentityService.URL
 	serviceUser = clusterConfig.Configure.Controller.IdentityUser
 	servicePassword = clusterConfig.Configure.Controller.IdentityPassword
+	keyringPath = clusterConfig.Configure.Storage.SecretPath
```

Comment:
I was actually thinking of completely removing the defaults in a later patch since I had the two fields being required for the min config but I wanted to get feedback first.